perception/lidar_centerpoint/lib/network/network_trt.cpp

```diff
@@ -59,6 +61,13 @@ bool HeadTRT::setProfile(
 
   for (std::size_t ci = 0; ci < out_channel_sizes_.size(); ci++) {
     auto out_name = network.getOutput(ci)->getName();
+
+    if (
+      out_name == std::string("heatmap") &&
+      network.getOutput(ci)->getDimensions().d[1] != static_cast<int32_t>(out_channel_sizes_[ci])) {
+      std::cout << "Expected and actual number of classes do not match" << std::endl;
```

Comment:
Cloud you try to use `logger_`? It's a wapper of ros message for tensort library.
https://github.com/autowarefoundation/autoware.universe/blob/main/common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp#L38-L73

Reply:
@yukke42 
Currently the caller also uses `std::cout`
`perception/lidar_centerpoint/lib/network/tensorrt_wrapper.cpp`

Should I replace all `std::cout` for consistency?

Reply:
@knzo25 I forget to replace them in the previous pr..., Cloud you replace others as well? :pray:

Reply:
Replaced the `std::cout` calls in [f0f0d38](https://github.com/autowarefoundation/autoware.universe/pull/3101/commits/f0f0d380370824f731d20277a5df5511f67d4d39)
Also fixed some typos